numpy/ma/core.py

```diff
@@ -7341,6 +7341,118 @@ def size(obj, axis=None):
 size.__doc__ = np.size.__doc__
 
 
+def diff(a, n=1, axis=-1, prepend=np._NoValue, append=np._NoValue):
```

Comment:
```suggestion
def diff(a, /, n=1, axis=-1, prepend=np._NoValue, append=np._NoValue):
```
Just thought we should start doing this, its not like `a` is a name that is nice enough we should promise it in all eternity.

I forgot one other thing (sorry about that):  Can you add this function to the stubs file in `core.pyi` (should be able to adapt the original `.pyi` file again).

Reply:
original pyi diff is this:

    @overload
    def diff(
        a: _T,
        n: L[0],
        axis: SupportsIndex = ...,
        prepend: ArrayLike = ...,
        append: ArrayLike = ...,
    ) -> _T: ...

but all declarations in numpy/ma/core.pyi look like this 

    def where(condition, x=..., y=...): ...
    def choose(indices, choices, out=..., mode=...): ...
    def round_(a, decimals=..., out=...): ...
    ...
Which should I follow?